components/brave_ads/browser/ads_service_factory.cc

```diff
@@ -77,9 +78,19 @@ bool AdsServiceFactory::ServiceIsNULLWhileTesting() const {
 void AdsServiceFactory::RegisterProfilePrefs(
     user_prefs::PrefRegistrySyncable* registry) {
   registry->RegisterBooleanPref(prefs::kBraveAdsEnabled, false);
+
   registry->RegisterUint64Pref(prefs::kBraveAdsPerHour, 2);
-  registry->RegisterUint64Pref(prefs::kBraveAdsPerDay, 6);
+
+  #if defined(OS_ANDROID)
+    const int ads_per_day = 12;
+  #else
+    const int ads_per_day = 20;
+  #endif
+  registry->RegisterUint64Pref(prefs::kBraveAdsPerDay, ads_per_day);
+
   registry->RegisterIntegerPref(prefs::kBraveAdsIdleThreshold, 15);
+
+  registry->RegisterIntegerPref(prefs::kBraveAdsPrefsVersion, 1);
```

Comment:
seems like it will be easy to forget to update this later. Maybe add `kBraveAdsPrefsDefaultVersion` so it's very obvious next to `kBraveAdsPrefsCurrentVersion`?

Reply:
or maybe kBraveAdsPrefsCurrentVersion should start at 1?

Reply:
Added `kBraveAdsPrefsDefaultVersion`, however cannot start current version at 1 as this would not migrate ads per day from 6 to the new value of 12 or 20.

Reply:
Added kBraveAdsPrefsDefaultVersion to make it more obvious